packages/marko-web-identity-x/browser/form/fields/email.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <form-group>
+    <label :for="id">{{ label }}</label>
+    <input
+      :id="id"
+      v-model="email"
+      class="form-control"
```

Comment:
Bootstrap styles... or are these supposed to be included in the `form.scss` file? (which is currently missing)